source/usage-examples/insertOne.txt

```diff
@@ -0,0 +1,91 @@
+.. _csharp-insert-one:
+
+=================
+Insert a Document
+=================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+You can insert a single document into a ``MongoCollection`` by using the
+`InsertOne() <{+api-root+}/M_MongoDB_Driver_IMongoCollection_1_InsertOne.htm>`__
+method.
```

Comment:
Since we also show the async method, maybe:
> You can insert a single document into ``MongoCollection`` by using the synchronous ... method, or the asynchronous ... method.

Reply:
Also, further down we say "collection", but here we say MongoCollection. I wonder if the specificity is needed?